sound/soc/intel/boards/Kconfig

```diff
@@ -580,7 +580,8 @@ config SND_SOC_INTEL_SOUNDWIRE_SOF_MACH
 	select SND_SOC_RT715_SDCA_SDW
 	select SND_SOC_RT5682_SDW
 	select SND_SOC_DMIC
-        help
+	select REGMAP_SOUNDWIRE_MBQ
```

Comment:
Shouldn't this selected by codec driver?

Reply:
yes, this was just a hack to make sure the code compiles. this is not intended to remain as is, the codec driver would select this if they need MBQ support